tests/python/LookTest.py

```diff
@@ -1,28 +1,173 @@
+        self.assertEqual(self.look.getName(), '')
+
+        for name in TEST_NAMES:
+            self.look.setName(name)
+            self.assertEqual(name, self.look.getName())
+
+        # Wrong type tests.
+        for invalid in (None, 1):
+            with self.assertRaises(TypeError):
+                self.look.setName(invalid)
+
+    def test_process_space(self):
+        """
+        Test the setProcessSpace() and getProcessName() methods.
+        """
+
+        # Default initialized process space value is ""
+        self.assertEqual(self.look.getProcessSpace(), '')
+
+        for process_space in self.TEST_PROCESS_SPACES:
+            self.look.setProcessSpace(process_space)
+            self.assertEqual(process_space, self.look.getProcessSpace())
+
+        # Wrong type tests.
+        for invalid in (None, 1):
+            with self.assertRaises(TypeError):
+                self.look.setProcessSpace(invalid)
+
+    def test_description(self):
+        """
+        Test the setDescription() and getDescription() methods.
+        """
+
+        # Default initialized description value is ""
+        self.assertEqual(self.look.getDescription(), '')
+
+        for desc in TEST_DESCS:
+            self.look.setDescription(desc)
+            self.assertEqual(desc, self.look.getDescription())
+
+        # Wrong type tests.
+        for invalid in (None, 1):
+            with self.assertRaises(TypeError):
+                self.look.setDescription(invalid)
+
+    def test_transform(self):
+        """
+        Test the setTransform() and getTransform() methods.
+        """
+
+        # Default initialized transform value is None
+        self.assertIsNone(self.look.getTransform())
+
+        exp_tr = OCIO.ExponentTransform()
+        exp_tr.setValue(self.TEST_EXP_VALUES)
+        self.look.setTransform(exp_tr)
+        out_exp_tr = self.look.getTransform()
+        self.assertListEqual(out_exp_tr.getValue(), self.TEST_EXP_VALUES)
+
+        # Wrong type tests.
+        for invalid in (OCIO.ALLOCATION_UNIFORM, 1):
+            with self.assertRaises(TypeError):
+                self.look.setTransform(invalid)
+
+    def test_inverse_transform(self):
+        """
+        Test the setInverseTransform() and getInverseTransform() methods.
+        """
+
+        # Default initialized inverse transform value is None
+        self.assertIsNone(self.look.getInverseTransform())
+
+        exp_tr = OCIO.ExponentTransform()
+        inv_exp_values = [1.0 / v for v in self.TEST_EXP_VALUES]
+        exp_tr.setValue(inv_exp_values)
+        self.look.setInverseTransform(exp_tr)
+        inv_oet = self.look.getInverseTransform()
+        self.assertListEqual(inv_oet.getValue(), inv_exp_values)
+
+        # Wrong type tests.
+        for invalid in (OCIO.ALLOCATION_UNIFORM, 1):
+            with self.assertRaises(TypeError):
+                self.look.setInverseTransform(invalid)
+
+    def test_constructor_with_positional(self):
```

Comment:
This should be `test_constructor_with_keyword` or `test_constructor_keyword`. The next method down should be `test_constructor_with_positional` or `test_constructor_positional`. Positional implies no keyword args are used, and that you are relying on the argument order defined in the function.